### PR TITLE
TC_002.001.004 | Radiant Tee product page > Visibility of the product description and detailed information

### PR DESCRIPTION
**Pull request by ElenaBarteneva**
<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;EXhjde1P">TC_002.001.004 | Radiant Tee product page &gt; Visibility of the product description and detailed information</a></blockquote>

### CODE REVIEW

**Comment by ElenaBarteneva** [comment]
[TC_002.001.004 | Radiant Tee product page > Visibility of the product description and detailed information](https://trello.com/c/EXhjde1P/107-tc002001004-radiant-tee-product-page-visibility-of-the-product-description-and-detailed-information)

**Review by YuliyaMihaylovaQA — APPROVED** [review]

File: pages/base_page.py [hunk 1]
```diff
@@ -108,6 +111,47 @@ def get_text(selector):
...
+    def add_product_to_cart_with_qty(self, size, color, qty):
+        s(f'[option-label={size}]').click()
+        s(f'[option-label={color}]').click()
+        s(PL.PRODUCT_QTY).click()
+        s(PL.PRODUCT_QTY).clear()
+        s(PL.PRODUCT_QTY).type(qty)
+        s(PL.ADD_TO_CART_BUTTON).click()
+        self.is_visible_success_message()
+
+    def is_minicart_subtotal_correct(self, qty):
```
Comment (Lokankara) [inline pages/base_page.py:140]: these method you can put into ` self.mini_card = MiniCard()`

File: pages/base_page.py [hunk 2]
```diff
@@ -108,6 +111,47 @@ def get_text(selector):
...
+
+    def delete_product_from_cart(self):
+        self.visit(CART_LINK)
+        s(Cart.REMOVE_ITEM_ICON).click()
+        s(Cart.NO_ITEMS_MESSAGE).wait_until(be.visible)
+
+    def add_product_to_cart_with_qty(self, size, color, qty):
+        s(f'[option-label={size}]').click()
+        s(f'[option-label={color}]').click()
+        s(PL.PRODUCT_QTY).click()
```
Comment (Lokankara) [inline pages/base_page.py:134]: you can create `product_qty = s(PL.PRODUCT_QTY)` below

**Review by denkho — APPROVED** [review]
great!

**Review by Lokankara — APPROVED** [review]

**Pull request merged**